docs/guides/javascript.rst

```diff
@@ -466,11 +466,17 @@ Plugins can trigger their own hooks:
 Available hooks
 ^^^^^^^^^^^^^^^
 
```

Comment:
@hypeJunction All the client-side hooks are documented separately here. In the future we'll want to expand the JS docs, but this seems OK for now.